source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift

```diff
@@ -118,6 +118,9 @@ class ACRColumnView: ACRContentStackView {
     }
     
     func configureColumnProperties(for view: NSView) {
+        if view is ACRDateField || view is ACRSingleLineInputTextView || view is ACRMultilineInputTextView {
+            minWidthConstraint.constant = 100
+        }
```

Comment:
I get why you are doing this, but what is the expected behaviour in microsoft for teams?

Reply:
in [adaptive card designer](https://adaptivecards.io/designer) got the following responses
 
<img width="407" alt="Screenshot 2022-09-15 at 9 36 44 AM" src="https://user-images.githubusercontent.com/105856097/190311940-746cf9f7-70bc-4ee5-ae9f-b1a32a0eb219.png">

<img width="408" alt="Screenshot 2022-09-15 at 9 36 10 AM" src="https://user-images.githubusercontent.com/105856097/190311965-c0d83ee7-d730-41f7-8fcf-4d3ea2611fc6.png">

<img width="407" alt="Screenshot 2022-09-15 at 9 36 26 AM" src="https://user-images.githubusercontent.com/105856097/190311987-e35e5cac-fc41-45e5-8c15-4e0fcabdd32f.png">